src/com/google/javascript/jscomp/CompilationLevel.java

```diff
@@ -163,7 +163,7 @@ private static void applyFullCompilationOptions(CompilerOptions options) {
     options.setRemoveAbstractMethods(true);
     options.setRemoveSuperMethods(true);
     options.setReserveRawExports(true);
-    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.ALL_UNQUOTED);
+    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.OFF);
```

Comment:
Was this intended to be included? Turning off property renaming isn't something we can just do - it's one of the biggest advantages of `ADVANCED` mode.